spec/features/site/user_visits_home_page_spec.rb

```diff
@@ -3,7 +3,7 @@
 feature 'User visits home page' do
   context 'when they are signed in' do
     background { sign_in_with(user.email, user.password) }
-    given(:user) { create(:user) }
+    given(:user) { create(:user, :moderator) }
```

Comment:
@LucasAnderson1984 In an ideal world you might want to try different contexts to test different roles, but to be honest it has minimum returns.

I wouldn't complain if you wanted to add them though :)